x-pack/plugins/data_enhanced/common/search/sessions_mgmt/date_string.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment';
+import { DATE_STRING_FORMAT } from './constants';
+
+export const dateString = (inputString: string, tz: string): string => {
```

Comment:
Originally I had it in UI. I moved it here per @lukasolson feedback: https://github.com/elastic/kibana/pull/81707#discussion_r547541345

I think it's good to have it in common since it seems like it will be useful in the future.

Reply:
On second glance, this code only makes sense in the browser, since it calls `moment.tz.guess()`

I'll move it back to public